espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/ElementValue.kt

```diff
@@ -80,6 +80,32 @@ class ElementValue(private val isReplacing: Boolean) : RequestHandler<TextValueP
             }
         }
 
-        return null
+        return
     }
-}
\ No newline at end of file
+
+    @Throws(AppiumException::class)
+    override fun handleCompose(params: TextValueParams): Unit {
+        val value: String = textToEnter(params)
+        try {
+            if (isReplacing) {
+                getNodeInteractionById(params.elementId).performTextClearance()
+            }
+            getNodeInteractionById(params.elementId).performTextInput(value)
+        } catch (e: IllegalArgumentException) {
+            throw InvalidElementStateException("Clear", params.elementId!!, e)
+        }
+    }
+
+    private fun textToEnter(params: TextValueParams) =
```

Comment:
usually method names must include verbs, for example, `parseTextToEnter` or `extractTextToEnter`